index.html

```diff
@@ -1498,6 +1515,60 @@
 
 <aside class="issue">Some services have the user pay for their use in data. These services aren't necessarily retaliating by denying their services to users who refuse to pay with data, but the details are more complex than we've had time to write.</aside>
 
+## Support Choosing Which Information to Present {#support-choosing-info}
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    [=User agents=] should support [=people=] in choosing which information they provide to [=parties=] that
+    request it, up to and including allowing for the provision of information that may not be factually
+    correct.
+  </span>
+</div>
+
+[=Parties=] can invest time and energy into automating ways of gathering [=data=] from [=people=] and can
+design their products in ways that make it a lot easier for [=people=] to disclose information than not, whereas
+[=people=] typically have to manually wade through options, repeated prompts, and deceptive patterns. In many
+cases, the absence of data — when a [=person=] refuses to provide some information — can also be identifying
+or revealing. Additionally, APIs can be defined or implemented in rigid ways that can prevent people from
+accessing useful functionality (eg. if my geolocation is forcefully set to match my GPS, a restaurant-finding
+site might restrict me to my current location when I want to look for restaurants in a city I will be visiting
+this weekend).
+
+[=User agents=] should make it simple for [=people=] to present the identity they wish to and to provide information
```

Comment:
"present the identify they wish to" is already covered by https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/191.html#principle-identity-per-context; do we need to repeat it here? Or we could link to that principle?

```suggestion
[=User agents=] should make it simple for [=people=] to [present the identity they wish to](#principle-identity-per-context) and to provide information
```

Reply:
+1 for linking to that existing principle